packages/extension-browser/src/devtools/views/pages/config/config-label.css

```diff
@@ -1,4 +1,11 @@
 .root {
     display: block;
-    margin: 0.75rem 1.25rem;
+    font-size: 1.166em;
+    line-height: 20px;
```

Comment:
Also should be in `rem` for consistency with the rest of the styling. Not sure we're ever going to be scaled in this context, but...